Pod/Classes/PINRemoteImageManager.m

```diff
@@ -225,11 +247,6 @@ - (instancetype)initWithSessionConfiguration:(NSURLSessionConfiguration *)config
     return self;
 }
 
-- (PINCache *)defaultImageCache;
-{
-    return [[PINCache alloc] initWithName:@"PINRemoteImageManagerCache"];
-}
-
```

Comment:
Would it make sense to leave this method in? The intention was to allow setting the cache that the shared instance uses. This way, if you wanted the shared instance to use a custom cache you could subclass and override this method.